router/pid_values.go

```diff
@@ -0,0 +1,11 @@
+package router
+
+import "github.com/AsynkronIT/protoactor-go/actor"
+
+type pidValues struct {
```

Comment:
I don't think a new type is necessary. It's simpler and more readable to just use `*[]*actor.PID`.